app/src/main/res/values/strings.xml

```diff
@@ -48,8 +48,8 @@
     <string name="login_forgot_password">Forgot password?</string>
     <string name="login_user_identifier_placeholder">jane@example.com or jane.doe</string>
     <string name="login_user_identifier_label">EMAIL OR USERNAME</string>
-    <string name="login_error_invalid_credentials_title">Invalid information</string>
-    <string name="login_error_invalid_credentials_message">Please validate your input and try again</string>
+    <string name="login_error_invalid_credentials_title">Login Failed</string>
+    <string name="login_error_invalid_credentials_message">The credentials you entered did not match any account. Please double-check and try again.</string>
```

Comment:
Previous texts were added based on the designs, Wolfgang should approve this
<img width="620" alt="image" src="https://user-images.githubusercontent.com/30429749/154233965-9e0b356e-f34c-46f1-8518-136ba0d6d5a7.png">

Reply:
Oh! Ok, I'll ask him for design review :D

Reply:
can we do something about it? 😄